### PR TITLE
added horoscope.py

### PR DESCRIPTION
**Pull request by ashish-patwal**
A python script that shows your horoscope based on your zodiac sign .

### CODE REVIEW

**Comment by ssm0801** [comment]
This is really a good one. Can you just add a small info about inquirer at top and are you participating in `hacktoberfest` ?

**Comment by ashish-patwal** [comment]
> This is really a good one. Can you just add a small info about inquirer at top and are you participating in `hacktoberfest` ?

Sure i can add info about inquirer package . This package was quite popular in node and js , turned out it had a clone for python too . So do you want me to add the data about inquirer at the top of horoscope,py ? And yes i'm participating in hacktoberfest .

File: Horoscope/horoscope.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    )
+    soup = BeautifulSoup(requests.get(url).content, "html.parser")
+    return soup.find("div", class_="main-horoscope").p.text
+
+
+if __name__ == "__main__":
+    questions = [
+        inquirer.List(
+            "sign",
+            message="Choose your Zodiac : ",
```
Comment (ashish-patwal) [inline Horoscope/horoscope.py:35]: we need to remove `:` as inquirer appends `:` to every message question

**Review by ssm0801 — APPROVED** [review]
Thanks for contribution !! 😄
I will merge it on 1st Oct so that it will be counted in `hacktoberfest`.

**Comment by ssm0801** [comment]
Add this kind of info at beginning of .py file

"""
    Text to Handwriting
    - Convert any text into handwriting and create image
    Author : Sudhanshu Motewar
    Date   : 24/09/21
"""

**Comment by ashish-patwal** [comment]
> Add this kind of info at beginning of .py file
> 
> """ Text to Handwriting - Convert any text into handwriting and create image Author : Sudhanshu Motewar Date : 24/09/21 """

done . PTAL @ssm0801

**Comment by ashish-patwal** [comment]
hey @ssm0801 . I am opening this script in other PR . Please merge that

**Pull request closed without merging**